mathesar/migrations/0017_auto_20210731_0107.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.7 on 2021-07-31 01:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mathesar', '0016_auto_20210728_2156'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='datafile',
+            name='created_from',
+            field=models.CharField(choices=[('FILE', 'File'), ('PASTE', 'Paste'), ('URL', 'Url')], max_length=128),
```

Comment:
Is there a way to put these choices in an Enum or another constant and then just expand them here? If you list choices out literally in migration files, every time a choice is added a new migration file is added, when it really doesn't need to.

This is just a suggestiont though, no need to change it if you feel otherwise!

Reply:
Didn't realize this was an option, thanks!